<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonGet">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>getData</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>260</width>
      <height>430</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_ip_servidor">
         <property name="text">
          <string>IP do Servidor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_ip">
         <property name="maxLength">
          <number>32770</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_connect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_disconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lista_ip"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_update">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_timing">
           <property name="text">
            <string>Timing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_timing">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_timing">
         <property name="intValue" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Plotter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_debug">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>debug</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_timing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_timing</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
